src/components/Quiz.js

```diff
@@ -18,19 +18,37 @@ const Quiz = () => {
     setQuestions(quizData);
   }, [quizName]);
 
-  const handleAnswerSelect = e => {
-    const selectedAnswer = { [e.target.name]: Number(e.target.value) };
+  const handleCheckboxAnswers = (e, answers) => {
+    const questionId = e.target.name;
+    const answerId = Number(e.target.value);
+    const previousSelectedAnswers = answers[questionId] || [];
+
+    const newCheckboxAnswer = previousSelectedAnswers.includes(answerId)
+      ? previousSelectedAnswers.filter(answer => answer !== answerId)
+      : [...previousSelectedAnswers.concat(answerId)];
```

Comment:
```suggestion
      : previousSelectedAnswers.concat(answerId);
```
`concat` returns a new array so we don't need to spread it into a new array

Reply:
Done!